include/dsn/utility/metrics.h

```diff
@@ -343,20 +375,30 @@ class counter : public metric
 
     virtual ~counter() = default;
 
+    long_adder_wrapper<Adder> _adder;
```

Comment:
why did you make `_addr` public?

Reply:
> why did you make `_addr` public?

Actually `_addr` is protected, see L373~